person_capture/video_io.py

```
@@ -2448,16 +2450,21 @@ def _chain(self) -> str:
             desat_key = self._lp_supports_any("desaturation", "desat")
             if desat_key:
                 lp_opts.append(f"{desat_key}={desat:.3f}")
             else:
                 sat_key = self._lp_supports_any("saturation", "sat")
                 if sat_key:
-                    # Desat=0 → saturation=1.0, desat=1 → saturation≈0
-                    sat = max(0.0, 1.0 - desat)
-                    lp_opts.append(f"{sat_key}={sat:.3f}")
+                    try:
+                        sat = float(os.getenv("PC_HDR_PREVIEW_GLOBAL_SATURATION", "1.0"))
+                    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)